src/app/content/externalClickHandler.ts

```diff
@@ -28,15 +28,27 @@ export const isHtmlElementInteractive = (element?: HTMLElement | null) =>
   element.matches &&
   interactiveElementsSelectors.some(selector => element.matches(selector));
 
+const isHtmlChromeElementInteractive = (chromePath?: string[] | null) =>
+  chromePath &&
+  interactiveElementsSelectors.some(selector =>
+    chromePath?.some(path => path === selector)
+  );
+
 export default (store: Store) => (e: MouseEvent) => {
   const state = store.getState();
 
   const element = e.target as HTMLElement;
   const parentElement = element.parentElement;
+  // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+  // @ts-ignore
+  const chromePath: string[] | null = e?.path && e.path.map(p => p.localName);
```

Comment:
Could you put this Chrome specific code in the `isHtmlChromeElementInteractive` function instead?
The function name might need a name and argument change....

Afther a bit of digging on this `path` atribute, there is actually a cross-browser solution see:
- https://stackoverflow.com/questions/39245488/event-path-is-undefined-running-in-firefox
- https://developer.mozilla.org/en-US/docs/Web/API/Event/composedPath

By using the cross-browser solution you might be able to simplify this to smthing like:
```
isEventPathInteractive(e.composePath);
```
or 
```
isEventInteractive(e);
```